### PR TITLE
[PL] add translation of collection

### PR DESCRIPTION
**Pull request by Koziolek**
in case of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @Koziolek!

@kdebowski could check this one too?  Thank you.

File: pl/basics/collections.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+```elixir
+iex> map = %{:foo => "bar", "hello" => :world}
+%{:foo => "bar", "hello" => :world}
+iex> map[:foo]
+"bar"
+iex> map["hello"]
+:world
+```
+
+Od wersji 1.2 Elixir 1.2 zmienne mogą być użyte jako klucz:
```
Comment (kdebowski) [inline pl/basics/collections.md]: Should be: 

> Od wersji 1.2 Elixira zmienne mogą być użyte jako klucz:

**Comment by kdebowski** [comment]
@Koziolek there is only one mistake. I added a note about that.

**Comment by Koziolek** [comment]
Done. 

@doomspork 
Ready to merge

**Comment by doomspork** [comment]
Thank you @Koziolek 👍

**Pull request merged**